packages/edge/android/src/main/java/com/adobe/marketing/mobile/reactnative/edge/RCTAEPEdgeArrayUtil.java

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+package com.adobe.marketing.mobile.reactnative.edge;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.bridge.ReadableType;
+import com.facebook.react.bridge.WritableArray;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class RCTAEPEdgeArrayUtil {
```

Comment:
Does the util class need to be public?

Reply:
We carry over the format from ACP react native and all the extensions are setting it as public right now.  We can raise a issue ticket to look into change this to protected later.

Reply:
As we discussed offline, let's revisit this and reduce the visibility for these files to package private if possible before the release.